pkg/appimport/appimport_test.go

```diff
@@ -52,16 +52,22 @@ func TestValidateAsset(t *testing.T) {
 	testArchivePath := filepath.Join(testdata, "somedb.sql.gz")
 	_, err = appimport.ValidateAsset(testArchivePath, "db")
 	assert.Error(err)
-	assert.Equal(err.Error(), "is archive")
+	if err != nil {
+		assert.Equal(err.Error(), "is archive")
+	}
```

Comment:
We'll want to wait until the import-files refactor PR is in and rebase this as the function signature for ValidateAsset has changed, it now returns a bool isArchive value and doesn't rely on the error value.